extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css

```diff
@@ -280,6 +280,10 @@ md-slider.md-default-theme.hue .md-thumb:after {
 	margin-top: -80px;
 }
 
+.card .extension-label {
+	max-width: 90%;
```

Comment:
it should only be restricted, if there is a docs url - otherwise, please use the available space.